animation_nodes/nodes/gpencil/gp_frame_info.py

```diff
@@ -0,0 +1,68 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... data_structures import GPStroke
+from ... base_types import AnimationNode, VectorizedSocket
+
+strokeTypeItems = [
+    ("ALL", "All Strokes", "Get all strokes", "NONE", 0),
+    ("INDEX", "Index Stroke ", "Get a specific stroke", "NONE", 1)
+]
+
+class GPFrameInfoNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPFrameInfoNode"
+    bl_label = "GP Frame Info"
+    errorHandlingType = "EXCEPTION"
+
+    strokeType: EnumProperty(name = "Stroke Type", default = "ALL",
+        items = strokeTypeItems, update = AnimationNode.refresh)
+
+    useIntegerList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("GPFrame", "Frame", "frame", dataIsModified = True)
```

Comment:
Is `dataIsModified` really needed here?

Reply:
It is not needed. I'll remove.

Reply:
Yes, it is needed for operation on the stroke level.